nibabel/cifti2/cifti2_axes.py

```diff
@@ -520,7 +520,7 @@ def to_cifti_brain_structure_name(name):
         ValueError: raised if the input name does not match a known anatomical structure in CIFTI-2
         """
         if name in cifti2.CIFTI_BRAIN_STRUCTURES:
```

Comment:
seems like this should be
```suggestion
        if name in cifti2.CIFTI_BRAIN_STRUCTURES.ciftiname:
```

Reply:
```python
>>> from nibabel.cifti2 import CIFTI_BRAIN_STRUCTURES
>>> 'CortexLeft' in CIFTI_BRAIN_STRUCTURES
True
>>> 'CortexLeft' in CIFTI_BRAIN_STRUCTURES.ciftiname
True
>>> CIFTI_BRAIN_STRUCTURES['CortexLeft']
'CORTEX_LEFT'
>>> CIFTI_BRAIN_STRUCTURES.name['CortexLeft']
'CORTEX_LEFT'
>>> CIFTI_BRAIN_STRUCTURES.ciftiname['CortexLeft']
'CIFTI_STRUCTURE_CORTEX_LEFT'
>>> CIFTI_BRAIN_STRUCTURES.guiname['CortexLeft']
'CortexLeft'
```

The idea here is that we want to accept any of these names, but then we'll map them out to the `ciftiname` form.